source/gravity_model/radial.cc

```diff
@@ -71,6 +77,11 @@ namespace aspect
         prm.leave_subsection ();
       }
       prm.leave_subsection ();
+      Assert (dynamic_cast<const GeometryModel::Box<dim>*> (&this->get_geometry_model()) == 0,
+              ExcMessage ("Gravity model Radial constant should not be used with geometry model Box."));
```

Comment:
Can you please add quotes around the names? Here and in other places, I don't care if they are single or double quotes.